test/models/user_test.rb

```diff
@@ -0,0 +1,41 @@
+require "test_helper"
+
+require 'pry'
+
+describe User do
+
+  before do
+    @user = User.create!(username: "Julia")
+    @saved_user = users(:first)
+    #User.first
+  end
+
+  describe 'validations' do
+    it 'is valid when a username is present' do
+
+      empty_hash = {}
+
+      expect(@saved_user.valid?).must_equal true
+      expect(@saved_user).must_be_kind_of User
+      expect(@saved_user.errors.messages).must_equal empty_hash
```

Comment:
Is this helpful to have a variable named `empty_hash` whose value is just `{}`? It would be better to use the minitest matcher `must_be_empty` because it's more evocative and it isn't checking that it's literally an empty hash